### ISSUE TITLE
[5:5] DRY up the Addresses used in our Specs for Organizations

### ISSUE THREAD

**Issue by armahillo** [loyno-mathcs/diaper#6]
# Summary 
This app uses a third-party Geocoding service (mapping an address to Long/Lat), which is hooked onto the `Organization` model. To ensure that we aren't spamming the service, we "stub" out the service, which means that we capture the expected outbound request and simulate a response using prepared data.

```
__________ App___________________________________________________3rd Party______
 [Organization] --> [Geocoder] --- Give me Long/Lat! -->      **MAGIC** 


__________ App___________________________________________________3rd Party______
 [Organization] <-- [Geocoder] --- 29.9792° N, 31.1342° E -- **MAGIC**
```
becomes
```
__________ App__________________________________________________________________3rd Party______
 [Organization] --> [Geocoder] --- Give me Long/Lat! --> [ *GOTCHA* ]             :'(            

__________ App__________________________________________________________________3rd Party______
 [Organization] <-- [Geocoder] <--- [ Fake Data ]                             :'''''(          

```

The problem is we have to be very specific about the address that's been watched for, in all of the attempts to stub the request. All outbound requests are intercepted, but the test will fail if there isn't a handler for it.

The stubs are defined in `rails_helper.rb`, in the `stub_addresses` method. `rails_helper` is executed before the test suite runs, just in time. (Ref: https://github.com/loyno-mathcs/diaper/blob/master/spec/rails_helper.rb#L61 )

We don't need all of these different addresses. We should only need 1 or 2, maybe 3 at most. (I think a couple of the tests required multiple addresses). We especially shouldn't have so many variations of the same address. 

One strategy for solving this might be:

 - Choose one address to keep, and comment out / delete the rest
 - Run the test suite
 - Identify each failed example and adjust the example, if possible, to use the address you already stubbed. If it *must* use a different address, then add that address back to the `stub_addresses` method

Be sure to consider the intent of what the spec is testing. The intent of the test has primacy over any nitpickery like this -- if the test requires the address to be provided in a particular way, and that way requires a different address to be stubbed, then that's ok.

Success would look like:
 - `stub_addresses` have a significantly fewer addresses defined
 - The test suite runs clean
 - The examples still test what they're supposed to

## Lifehack
Running the full test suite can be time-consuming. On a reasonably fast computer it will still take a few minutes to run fully.

However, after you run it once, if there are any failures, those are saved in a file, and you can tell RSpec to only run those files by using:

```bash
$ bundle exec rspec --only-failures
```

This will only run the examples that failed last time you ran the suite. If you really want to narrow it down (sometimes this is helpful when you're grinding through a bunch of failed specs), then try:

```bash
$ bundle exec rspec --next-failure
```

**Comment by nihonjinrxs** [comment]
@jjlahatte I see you've move this to "Needs review" on the project board. Is there a pull request associated with this issue? If so, please comment on that in the PR using the issue number (i.e. `#6`) to link the two. If not, please move it back to in progress until there is a PR available for review.

**Comment by jjlahatte** [comment]
yes there is a pull request associated with this issue already

**Comment by nihonjinrxs** [comment]
Resolved on upstream PR rubyforgood/diaper#724, merged to master at a4a9b4ae04a3354fb4c2b519c619778b6c9b8c8d.
